build.gradle.kts

```diff
@@ -142,10 +191,15 @@ subprojects {
     apply(plugin = "com.adarshr.test-logger")
     apply(plugin = "io.freefair.lombok")
 
-    extra["lombokVersion"] = extensions.findByType(LombokExtension::class)?.version?.get()
-    tasks.named("generateLombokConfig") {
-        // Don't generate lombok.config files ( https://docs.freefair.io/gradle-plugins/3.6.6/reference/#_lombok_config_handling )
-        enabled = false
+    extra["lombokVersion"] = extensions.findByType(LombokExtension::class)?.version
```

Comment:
I am not sure if the `?` is strictly necessary here. I mean, we load the Lombok extension on all Gradle versions (we can't build without it).

I had changed it from `.version?.get()` to `version` for older versions.

Reply:
Ok cool